### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.10 es 4.72 (13 novembre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#324] [Esercizi M-Z]
![immagine](https://github.com/user-attachments/assets/38a919d9-fc99-4251-a7d6-ebcf27ab2daa)

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/004b7e3d-dfda-41cf-ab05-2c9e9194a1a7)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/fbcc9748-5f00-4d44-b013-2abab187269f)
![immagine](https://github.com/user-attachments/assets/805510b2-23b5-4bea-bd4c-21d79ef40157)
